doc/README.md

```diff
@@ -0,0 +1,118 @@
+<dd></dd>
+</dl>
+
+<a name="Smartcar"></a>
+
+## Smartcar
+**Kind**: global class
+
+* [Smartcar](#Smartcar)
+    * [new Smartcar(options)](#new_Smartcar_new)
+    * _instance_
+        * [.getAuthUrl(options)](#Smartcar+getAuthUrl) ⇒ <code>String</code>
+        * [.openDialog(options)](#Smartcar+openDialog) ⇒ <code>String</code>
+        * [.addClickHandler(options)](#Smartcar+addClickHandler)
+    * _static_
+        * [.AccessDenied](#Smartcar.AccessDenied) ⇐ <code>Error</code>
+            * [new AccessDenied(message)](#new_Smartcar.AccessDenied_new)
+
+<a name="new_Smartcar_new"></a>
+
+### new Smartcar(options)
+Initializes Smartcar class
+
+
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| options | <code>Object</code> |  | the SDK configuration object |
+| options.clientId | <code>String</code> |  | the application's client id |
+| options.redirectUri | <code>String</code> |  | the registered redirect uri of the application |
+| [options.scope] | <code>Array.&lt;String&gt;</code> |  | requested permission scopes |
+| [options.onComplete] | [<code>OnComplete</code>](#OnComplete) |  | called on completion of auth flow |
+| [options.development] | <code>Boolean</code> | <code>false</code> | launch Smartcar auth in development mode to enable the mock vehicle brand |
+
+<a name="Smartcar+getAuthUrl"></a>
+
+### smartcar.getAuthUrl(options) ⇒ <code>String</code>
+Generates Smartcar OAuth URL
+
+**Kind**: instance method of [<code>Smartcar</code>](#Smartcar)
+**Returns**: <code>String</code> - generated OAuth link
+
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| options | <code>Object</code> |  | the link configuration object |
+| [options.state] | <code>String</code> |  | arbitrary parameter passed to redirect uri |
+| [options.forcePrompt] | <code>Boolean</code> | <code>false</code> | force permission approval screen to show on every authentication, even if the user has previously consented to the exact scope of permission |
+
+<a name="Smartcar+openDialog"></a>
+
+### smartcar.openDialog(options) ⇒ <code>String</code>
+Launch the OAuth dialog flow
+
+**Kind**: instance method of [<code>Smartcar</code>](#Smartcar)
+**Returns**: <code>String</code> - generated OAuth link
+
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| options | <code>Object</code> |  | the link configuration object |
+| [options.state] | <code>String</code> |  | arbitrary parameter passed to redirect uri |
+| [options.forcePrompt] | <code>Boolean</code> | <code>false</code> | force permission approval screen to show on every authentication, even if the user has previously consented to the exact scope of permission |
+
+<a name="Smartcar+addClickHandler"></a>
+
+### smartcar.addClickHandler(options)
+Add an on-click event listener to the element with the provided id.
+On-click event calls openDialog when the specified element is clicked.
```

Comment:
On-click event calls openDialog when the specified element is **clicked** (no period)